src/util/LineChart/index.js

```diff
@@ -106,11 +106,7 @@ export default class LineChart {
 
     this.yAxis
       .transition()
-      .call(
-        axisLeft(this.y).tickFormat((d) =>
-          formatKPIValue(kpi, d, { compact: true })
-        )
-      )
+      .call(axisLeft(this.y).tickFormat((d) => formatKPIValue(kpi, d)))
```

Comment:
Kan ikke potensielt ha høyere tallverdier? Må kanskje tas høyde for, også i PNG-eksporten.
<img width="152" alt="image" src="https://user-images.githubusercontent.com/2866225/195329281-eacfee83-c5aa-4767-ba93-0f857da857ac.png">

Reply:
> Kan ikke potensielt ha høyere tallverdier? Må kanskje tas høyde for, også i PNG-eksporten. <img alt="image" width="152" src="https://user-images.githubusercontent.com/2866225/195329281-eacfee83-c5aa-4767-ba93-0f857da857ac.png">

Det er for så vidt allerede et problem for prosentvisningen:

![image](https://user-images.githubusercontent.com/3673262/195344867-f797297b-3c71-4f0c-a364-7df39c46b556.png)

Men ja, burde kanskje ha en cutoff der sånn at man forkorter hvis tallet ellers ville bli kuttet bort. Et nytt middels kompakt format kanskje? Kan teste litt.